packages/cli/src/download/uss-file/UssFile.handler.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import * as fs from "node:fs";
+import { homedir } from "node:os";
+import type { IHandlerParameters } from "@zowe/imperative";
+import { IO } from "@zowe/imperative";
+import { B64String, type ZSshClient, type uss } from "zowe-native-proto-sdk";
+import { SshBaseHandler } from "../../SshBaseHandler";
+import path = require("node:path");
+
+export default class DownloadUssFileHandler extends SshBaseHandler {
+    public async processWithClient(params: IHandlerParameters, client: ZSshClient): Promise<uss.ReadFileResponse> {
+        const response = await client.uss.readFile({
+            fspath: params.arguments.filePath,
+            encoding: params.arguments.binary ? "binary" : params.arguments.encoding,
+        });
+        const content = B64String.decode(response.data);
+        const match = params.arguments.filePath.match(/[^/]+$/)?.[0] || "";
```

Comment:
Will this regex work for Windows systems? We might have to use `path.sep` to determine what character to place in the negated character class, e.g.:

```
const re = new RegExp(`/[^${path.sep === "\\" ? "\\\\" : "\"}+$/`);
```

Or if you'd prefer, you could save off the `filePath` into a string and replace all Windows path separators `\\` with the POSIX path separator `/`.